contracts/0.8.25/vaults/Delegation.sol

```diff
@@ -346,6 +348,20 @@ contract Delegation is Dashboard {
         _voluntaryDisconnect();
     }
 
+    /**
+     * @notice Pauses deposits to beacon chain from the StakingVault.
+     */
+    function pauseBeaconDeposits() external onlyRole(CURATOR_ROLE) {
```

Comment:
`Beacon deposits` seems a new thing for me. It was  `Beacon Chain deposits` before. Can confuse with beacon proxy. From the top of my mind:
- just deposits (but can be not specific enough) 
- BeaconChainDeposits (seems fine and synced with `makeBeaconChainDeposits` method  )
- ValidatorDeposits (will require to sync the naming over all codebase)

Reply:
fixed in https://github.com/lidofinance/core/commit/48bccda5b9e2ac126440877ee587c57f5518c2db